aggregator/flush_mgr.go

```diff
@@ -304,3 +308,15 @@ func newBucket(interval time.Duration, scope tally.Scope) *flushBucket {
 func (b *flushBucket) Add(flusher PeriodicFlusher) {
 	b.flushers = append(b.flushers, flusher)
 }
+
+func (b *flushBucket) Remove(flusher PeriodicFlusher) error {
+	numFlushers := len(b.flushers)
+	for i := 0; i < numFlushers; i++ {
+		if b.flushers[i] == flusher {
+			b.flushers[i], b.flushers[numFlushers-1] = b.flushers[numFlushers-1], b.flushers[i]
+			b.flushers = b.flushers[:numFlushers-1]
```

Comment:
Flushers don't async flush themselves yeah? Just checking we don't need to do anything with the flushers like close them or anything (I don't see anything in the interface)

Reply:
Yeah flushers are passive and their flushing is controlled by the flush manager. There is no need to close them or clean them up. These flushers are essentially lists storing aggregated metrics so they shouldn't be closed after flushing anyway.